test/realm/realm.conf

```diff
@@ -2,7 +2,7 @@
 basedn = dc=demo1,dc=freeipa,dc=org
 realm = DEMO1.FREEIPA.ORG
 domain = demo1.freeipa.org
-server = ipa.demo1.freeipa.org
+server = ipa-realm.demo1.freeipa.org
```

Comment:
What's this change for?

Anyway I am good, please rebase and we can merge.

Reply:
Because previously this triggered a failure in parsing. It would assume this was the option `realm = ...` instead of `server = ...`.